src/librustc_mir/interpret/intrinsics.rs

```diff
@@ -368,45 +368,19 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         &mut self,
         instance: ty::Instance<'tcx>,
         args: &[OpTy<'tcx, M::PointerTag>],
-        _ret: Option<(PlaceTy<'tcx, M::PointerTag>, mir::BasicBlock)>,
+        span: Span,
```

Comment:
Looks like we usually have `span` before `instance` and `args` (at least for `find_mir_or_eval_fn` and `call_intrinsic`); please make that consistent.